components/filter/filter-dimension-set.js

```diff
@@ -18,6 +18,15 @@ class FilterDimensionSet extends LitElement {
 			 * Whether the values for this dimension are still loading and a loading spinner should be displayed
 			 */
 			loading: { type: Boolean },
+			/**
+			 * Whether to hide the search input, fire an event on search, or perform a simple text search
+			 * @type {'none'|'event'|'simple'}
+			 */
+			searchType: { type: String, attribute: 'search-type' },
+			/**
+			 * Value for the search input
+			 */
+			searchValue: { type: String, attribute: 'search-value' },
```

Comment:
Is there a use case for them needing to specify an initial value to search on? Seems a bit weird.

Reply:
So the old labs filter allows this, but from my searching it doesn't _seem_ like it's been used anywhere.  I think the plan was to use it for Quick Eval, but it was decided to allow the filters to be set up in a default state on page load, but have the search still clear.  So I could probably remove this and keep things simple for now

Reply:
Right. Maybe for like a "saved search" type scenario where the search filters are reflected in the URL (for bookmarking, sending links) and then the state would need to be restored that way.